test/pool_SUITE.erl

```diff
@@ -23,6 +23,8 @@
 all() ->
 	[
 		checkout_checkin,
+		checkin_unknown,
+		checkin_not_owner,
```

Comment:
Do the same tests for `give_away` maybe?

Reply:
Done, and removed some unnecessary parts.